generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -122,7 +126,24 @@ export class <%= entityAngularName %>DialogComponent implements OnInit {
             });
         }
     }
-   <%_ } _%>
+
+    <%_ if (fieldsContainImageBlob) { _%>
+    clearInputImage(field: string, fieldContentType: string, idInput: string) {
```

Comment:
since this method is common may be this can be moved to data-util file and be called from here, may be even pass elementRef